test/defaults/with-config-and-entry/default-with-config.test.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const { stat } = require('fs');
+const { resolve } = require('path');
+const { run, extractSummary } = require('../../utils/test-utils');
+
+describe('output flag defaults with config', () => {
+    it('should use default entry if config entry file is not present', done => {
```

Comment:
I don't understand this test. Here there is a config file, so why the test says "not present"?

Reply:
The entry file specified in the config doesn't exist, so it will pick the default entry file `./index.js`.

Should I improve the wording here?

Reply:
Maybe `.. if entry file from config is not present` ?

Reply:
Yes, we should improve the wording: "should override the [entry] property of the config file" We don't care if the file really exists or not

Reply:
Actually, if the entry file exists, it picks up the entry specified in the config instead of the default entry but in case it doesn't then it falls back to using the default entry

Reply:
This should NOT be the case. The point of enforcing defaults is to use OUR defaults and override everything that already exists.

Reply:
Which means there's a bug in our strategy when we apply the defaults

Reply:
Yes, will take a look at that too. 👍

Reply:
Skipping this test for now, will enable it once I fix the above problem.